client/src/docs-ui/component-playground/custom-components/authenticator-with-slots.tsx

```diff
@@ -0,0 +1,30 @@
+import {h} from "@stencil/core";
+
+export const AuthenticatorWithSlots = () => (
```

Comment:
```diff
- import {h} from "@stencil/core";
+ import {h, FunctionalComponent} from "@stencil/core";

- export const AuthenticatorWithSlots = () => (
+ export const AuthenticatorWithSlots: FunctionalComponent = () => (
```

Reply:
A `FunctionalComponent` requires props. We don't need props for this.